### PR TITLE
update dependency's in linux build documents

### PR DESCRIPTION
**Pull request by savoptik**
I updated the [dependencies](https://github.com/RHVoice/RHVoice/issues/382) in the document, as the ones listed there are not enough to successfully build the project.

### CODE REVIEW

**Comment by alex19EP** [comment]
I would like these documents to be distro agnostic.

or we need to rewrite the instructions with a section for each distribution, but I don't like that.

File: doc/en/Compiling-on-Linux.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ audio libraries is installed on your machine:
 * [PulseAudio](https://www.freedesktop.org/wiki/Software/PulseAudio/)
 * [Libao](https://www.xiph.org/ao/)
 * [PortAudio](http://www.portaudio.com) V19
+* [libspeechd-dev](https://packages.debian.org/stretch/libspeechd-dev)
```
Comment (CyberTailor) [inline doc/en/Compiling-on-Linux.md:19]:
```suggestion
* [Speech Dispatcher](https://freebsoft.org/speechd)
```

File: doc/ru/Compiling-on-Linux.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
 * [PulseAudio](https://www.freedesktop.org/wiki/Software/PulseAudio/)
 * [Libao](https://www.xiph.org/ao/)
 * [PortAudio](http://www.portaudio.com) V19
+* [libspeechd-dev](https://packages.debian.org/stretch/libspeechd-dev)
```
Comment (CyberTailor) [inline doc/ru/Compiling-on-Linux.md:21]:
```suggestion
* [Speech Dispatcher](https://freebsoft.org/speechd)
```

**Review by CyberTailor — CHANGES_REQUESTED** [review]

**Comment by savoptik** [comment]
@alex19EP @CyberTailor  Colleagues, please don't forget to update the documentation.

**Pull request merged**